app/assets/stylesheets/common/_constants.scss

```diff
@@ -1,29 +1,63 @@
+:root {
+--primary_one: #245185;
+--primary_two: #cee3ea;
+--primary_three: #4c7c51;
+--background_main: #fff;
+--background_support: #cee3ea;
+--line: #464646;
+--sharp_line: #000;
+--heavy_error: #a20000;
+--error_light: #ffc2c2;
+--heavy_alert: #ffd452;
+--annotation_holder: #ffd452;
+--light_alert: #ffe9ac;
+--heavy_success: #246700;
+--light_success: #b9e586;
+
+--comments_color: var(--heavy_success);
+--key_words_color: var(--primary_one);
+--strings_color: var(--heavy_error);
+
+--darker-orange: #ee6415;
+}
+// Code viewer
+$comments: var(--comments_color);
+$key-words: var(--key_words_color);
+$strings: var(--strings_color);
+
+// General contexts
+$links: var(--primary_one);
+$pane-highlight: var(--primary_two);
+$confirmed: var(--light_success);
+$annotation-holder: var(--annotation_holder);
+
+// Status
+$severe-error: var(--heavy_error);

```

Comment:
Dividing into sections is good; within each section, order variables alphabetically.